plugins/weida-raw/fu-weida-raw.rs

```diff
@@ -19,3 +31,9 @@ enum FuWeidaRawCmd8760 {
     CALCULATE_FLASH_CHECKSUM = 0xD5,
 }
 
+#[repr(u16)]
+enum FuWeidaRawCmd8760u16 {
+    UNPROTECT_LOWER508K = 0x0044,
+    PROTECT_ALL = 0x007C,
```

Comment:
Weirdly, you specify the enums in CamelCase (which get converted to UPPER_CASE when we generate the header file) so these should be `UnprotectLower508k` and `ProtectAll`.

Reply:
solved